### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by TaiwoEnoch**
## I create this PR to feature the following listed below ⬇️ 

- [ ] Set up React Testing Library.
- [ ] Follow the instructions in the official [documentation](https://testing-library.com/docs/react-testing-library/intro).
- [ ] You don't need to setup Jest (it's included in Create React App).
- [ ] Write unit tests for the files operate.js and calculate.js using Jest.
- [ ] Create unit tests for all React components:
- [ ] Use Jest snapshots to test the components.
- [ ] Use React Testing Library to simulate user interaction.
- [ ] Run your full tests suite using Jest. All tests should pass.

### CODE REVIEW

File: src/logic/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  });
+
+  test('multiply a positive number with a negative mumber', () => {
+    expect(operate(1, -2, 'x')).toBe('-2');
+  });
+
+  test('test division operation', () => {
+    expect(operate(10, 2, '÷')).toBe('5');
+  });
+});
```
Comment (Lucas-Masaba) [inline src/logic/operate.test.js]: - [ ] Please add test(s) for the modulous `%` operator as well

File: src/logic/calculate.test.js
Comment (Lucas-Masaba) [inline src/logic/calculate.test.js:1]: [OPTIONAL]
- I would suggest having your test files in their own folder for example a folder named `tests`

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @TaiwoEnoch and @ab-noori  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Passing tests :heavy_check_mark: 
- Linter checks passing :heavy_check_mark: 
- Professional READrME :notebook: 
- Professional pull request message :scroll: 

## Required Changes ♻️
- [ ] ***Please write unit tests for all the React components you have in this project. Use Jest snapshots to test the components and React Testing Library to simulate user interaction. :+1: ***

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @TaiwoEnoch , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] All tests cases are running :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**